app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if @review.errors.any? %>
+<div id="error_explanation">
+  <h2><%= pluralize(@review.errors.count, "error") %> prohibited this review from being saved:</h2>
+  <ul>
+    <% @review.errors.full_messages.each do |message| %>
+    <li><%= message %></li>
+    <% end %>
+  </ul>
+</div>
+<% end %>
```

Comment:
It's fine to use either `<% end %>` or  `<%-end-%>` but remain consistent throughout your code.